interfaces/builtin/audio_playback_test.go

```diff
@@ -129,6 +129,9 @@ func (s *AudioPlaybackInterfaceSuite) TestSecCompOnClassic(c *C) {
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), Not(testutil.Contains), "owner /run/user/[0-9]*/snap.audio-playback/pulse/ r,\n")
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), Not(testutil.Contains), "owner /run/user/[0-9]*/snap.audio-playback/pulse/native rwk,\n")
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), Not(testutil.Contains), "owner /run/user/[0-9]*/snap.audio-playback/pulse/pid r,\n")
+	c.Assert(spec.SnippetForTag("snap.consumer.app"), Not(testutil.Contains), "owner /var/snap/snap.audio-playback/common/pulse/r,\n")
```

Comment:
Wait, something is weird here: the test function is `TestSecCompOnClassic` The whole code block above this is bogus as those are apparmor rules in a seccomp snippet.

Reply:
You are right. Fixed that, and also sent a PR to fix the other cases https://github.com/canonical/snapd/pull/15241